android/java/org/chromium/chrome/browser/crypto_wallet/SwipeOffViewPager.java

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.crypto_wallet;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+
+import androidx.viewpager.widget.ViewPager;
+
+public class SwipeOffViewPager extends ViewPager {
```

Comment:
@SergeyZhukovsky did you try using NonSwipaableViewPager from custom_layout ? it should work the same .

Reply:
cool, I missed that. Yeah there is not reason to create another class.